product-service/serverless.ts

```diff
@@ -219,8 +241,71 @@ const serverlessConfiguration: Serverless = {
           }
         }
       ]
+    },
+    catalogBatchProcess: {
+      handler: 'handler.catalogBatchProcess',
+      events: [
+        {
+          sqs: {
+            batchSize: 5,
+            arn: {
+              'Fn::GetAtt': ['SQSQueue', 'Arn']
+            }      
+          }
+        }
+      ]
+    }
+  },
+  resources: {
+    Resources: {
+      SQSQueue: {
+        Type: 'AWS::SQS::Queue',
+        Properties: {
+          QueueName: 'catalog-items-queue.fifo',
+          FifoQueue: true
+        }
+      },
+      SNSTopic: {
+        Type: 'AWS::SNS::Topic',
+        Properties: {
+          TopicName: 'create-product-topic',
+        }
+      },
+      SNSSubscription: {
+        Type: 'AWS::SNS::Subscription',
+        Properties: {
+          Endpoint: 'diamonddragontp@gmail.com',
+          Protocol: 'email',
+          TopicArn: {
+            Ref: 'SNSTopic'
+          },
+          FilterPolicy: {
+            "price": [
+              "10",
+              "2",
+              "4"
+            ]
```

Comment:
🙂

Reply:
Just to show the work of the filters for sns. Why don't send some messages with prices 10, 2 and 4 to some email maybe client asked me to do it.

Reply:
![image](https://user-images.githubusercontent.com/14991709/100096047-835a2200-2e6c-11eb-8ca9-3b39d6f224f2.png)
https://github.com/DzmitryU/rs-aws-be/pull/5/files#diff-bbe99e6bf77e85e88f14980601f1a819286338ad9bad2343f12ab90f8814d2d0
just for learning purposes